xbmc/interfaces/Builtins.cpp

```diff
@@ -1851,6 +1852,10 @@ int CBuiltins::Execute(const std::string& execString)
   {
     g_application.StopPVRManager();
   }
+  else if (execute == "pvr.searchmissingchannelicons")
+  {
+    g_application.SearchMissingChannelIcons();
```

Comment:
So basically I just call TriggerSearchMissingChannelIcons(); without checking first if the pvrManager has been started or do you mean to keep the verification but drop only the log ine?